vinyldns.go

```diff
@@ -142,6 +142,58 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:        "zone-create",
+			Usage:       "zone-create --name <name> --email <email> --admin-group-id <adminGroupID> --transfer-connection-name <transferConnectionName> --transfer-connection-key <transferConnectionKey> --transfer-connection-key-name <transferConnectionKeyName> --transfer-connection-primary-server <transferConnectionPrimaryServer> --zone-connection-name <zoneConnectionName> --zone-connection-key <zoneConnectionKey> --zone-connection-key-name <zoneConnectionKeyName> --zone-connection-primary-server <zoneConnectionPrimaryServer>",
+			Description: "Create a zone",
+			Action:      zoneCreate,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "name",
+					Usage: "The zone name",
+				},
+				cli.StringFlag{
+					Name:  "email",
+					Usage: "The zone email",
+				},
+				cli.StringFlag{
+					Name:  "admin-group-id",
+					Usage: "The zone admin group ID",
+				},
+				cli.StringFlag{
+					Name:  "transfer-connection-name",
+					Usage: "The zone transfer connection",
+				},
+				cli.StringFlag{
```

Comment:
should note that the connection and transfer connection sets of fields are optional.  We assume a global default.  Most users do not (and really should not) have their own connections.

As long as the cli does not mandate the fields, we are good.

Reply:
Gotcha; thanks for the info.